site/docs/api/configuration/parquet-mdstore-configuration.md

```diff
@@ -74,25 +74,23 @@ You should set the `io.xskipper.parquet.mdlocation` in one of two ways:
 === "Scala"
 
     ``` scala
-    from xskipper import Xskipper
-    
-    # The base location to store all indexes 
-    # TODO: change to your index base location
-    md_base_location = "/tmp/metadata"
-    
-    # Configuring the JVM wide parameters
-    conf = dict([
-                ("io.xskipper.parquet.mdlocation", md_base_location),
-                ("io.xskipper.parquet.mdlocation.type", "EXPLICIT_BASE_PATH_LOCATION")])
-    Xskipper.setConf(spark, conf)
+    import io.xskipper._
+    // TODO: Change to your location
+    val location = "/tmp/metadata"
+    val conf = Map(
+      "io.xskipper.parquet.mdlocation" -> location,
+      "io.xskipper.parquet.mdlocation.type" ->  "EXPLICIT_BASE_PATH_LOCATION")
+    // set JVM-wide parameters
+    Xskipper.setConf(conf)
+
     ```
 
-### Setting an explicit metadata location for Xskipper instance
+### Setting a specific metadata location for an Xskipper instance
```

Comment:
I think we should leave it as `explicit` since this is the way the config is written